pkg/api/errors/errors.go

```diff
@@ -226,7 +226,7 @@ func NewServerTimeout(kind, operation string, retryAfterSeconds int) error {
 		Details: &unversioned.StatusDetails{
 			Kind:              kind,
 			Name:              operation,
-			RetryAfterSeconds: retryAfterSeconds,
+			RetryAfterSeconds: int32(retryAfterSeconds),
```

Comment:
Agree, but I'd rather refactor the internal in a follow on (remove casts to
int32)

On Wed, Nov 18, 2015 at 6:04 PM, Daniel Smith <notifications@github.com>
wrote:

> In pkg/api/errors/errors.go
> <https://github.com/kubernetes/kubernetes/pull/17353#discussion_r45275000>
> :
>
> > @@ -226,7 +226,7 @@ func NewServerTimeout(kind, operation string, retryAfterSeconds int) error {
> >  		Details: &unversioned.StatusDetails{
> >  			Kind:              kind,
> >  			Name:              operation,
> > -			RetryAfterSeconds: retryAfterSeconds,
> > +			RetryAfterSeconds: int32(retryAfterSeconds),
>
> I don't expect it makes a difference for this field, but casting like this
> is not safe in general as it potentially hides a cross-platform bug.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17353/files#r45275000>.
>